pkg/api/defaults-controller-manager_test.go

```diff
@@ -196,7 +196,7 @@ func TestControllerManagerConfigFeatureGates(t *testing.T) {
 	// test user-overrides, removal of feature gates for k8s versions >= 1.28
 	cs = CreateMockContainerService("testcluster", defaultTestClusterVer, 3, 2, false)
 	cs.Properties.OrchestratorProfile.OrchestratorVersion = "1.28.0"
-	cs.Properties.OrchestratorProfile.KubernetesConfig.APIServerConfig = make(map[string]string)
+	cs.Properties.OrchestratorProfile.KubernetesConfig.ControllerManagerConfig = make(map[string]string)
```

Comment:
did you mean this? replacing APIServerConfig with ControllerManagerConfig?

Reply:
Yes, this unit test is for controller manager, I think this was a mistake from last time.